### DISCUSSION TITLE
How to fix: WalletLoginError: Failed to connect with walletAlready connected

### DISCUSSION THREAD

**Discussion by smartdev0530** [Web3Auth/Web3Auth#931] [Help]
I've created custom auth, and logged in with jwt.
When I re-login, this error occurs.
![image](https://user-images.githubusercontent.com/91709558/203855006-7ef0bf68-b447-49e3-8604-84ddb9eab929.png)

**Comment by seakkus** [comment]
Try this, that should solve the problem:

`    if (web3auth.status !== ADAPTER_STATUS.CONNECTED) {
        await web3auth.connectTo(openloginAdapter.name, {
            loginProvider: "jwt",
            extraLoginOptions: {
                id_token: idToken,
                verifierIdField: "sub",
                domain: window.origin
            },
        });
    }`
